release-please-config.json

```diff
@@ -10,5 +10,6 @@
     "gax": {},
     "tools": {}
   },
+  "include-v-in-tag": false,
```

Comment:
Actually, if you want plain `v1.2.3` tags, you want this to be `include-component-in-tag: false`